src/engine/v3/V3Interpreter.v3

```diff
@@ -35,6 +35,19 @@ class V3Interpreter extends WasmStack {
 		thrown = null;
 		state_ = if(params_arity == 0, StackState.RESUMABLE, StackState.SUSPENDED);
 	}
+	def forceOp(it: BytecodeIterator) {
```

Comment:
This will also need the stidetable pointer (stp) to be up to date. There's a way to get the proper sidetable position for a given PC using a reverse lookup in `SidetableMap`.